plugin/cmd/kube-scheduler/scheduler.go

```diff
@@ -33,14 +30,11 @@ func init() {
 
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
-	s := app.NewSchedulerServer()
-	s.AddFlags(pflag.CommandLine)
 
-	util.InitFlags()
+	cmd := app.NewSchedulerServerCommand()
+
 	util.InitLogs()
 	defer util.FlushLogs()
 
-	verflag.PrintAndExitIfRequested()
-
-	s.Run(pflag.CommandLine.Args())
+	cmd.Execute()
```

Comment:
I'm not sure what would cause a cobra command to return an error without calling s.Run but glog.Fatal(cmd.Execute()) to be safe?